seed/challenges/jquery.json

```diff
@@ -733,11 +733,11 @@
         "You've seen why id attributes are so convenient for targeting with jQuery selectors. But you won't always have such neat ids to work with.",
         "Fortunately, jQuery has some other tricks for targeting the right elements.",
         "jQuery uses CSS Selectors to target elements. <code>target:nth-child(n)</code> css selector allows you to select all the nth element with the target class or element type.",
-        "Make the second child in each of your well elements bounce.",
+        "Make the second child in each of your well elements bounce. You must target the children of the <code>target</code> (NOT THE <code>well</code>) class.",
```

Comment:
@awesomeaniruddh that's too explicit in specifying what to do and what not to do. :P